.editorconfig

```diff
@@ -0,0 +1,26 @@
+# http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
```

Comment:
Switching space to tab brings all sort of errors since editors are using tabs differently. I think it would be a good idea to switch back to space in our eslint too.